server/server_test.go

```diff
@@ -7751,6 +7756,7 @@ stream
 		.message('message')
 		.details('details')
 		.crit(lambda: TRUE)
+		.sensu()
```

Comment:
This bit of code here should be registering a sensu handler for the topic `test`. So it should get an alert event, this hasn't changed in your code so I am not sure why you would be getting that error. But it should definitely be passing without the `.sensu()` bit as it was before.